Vagrantfile

```diff
@@ -2,38 +2,45 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/trusty64"
+  config.vm.box = "ubuntu/xenial64"
 
   config.vm.provider "virtualbox" do |v|
     v.memory = 4096
     v.cpus = 2
   end
 
   config.vm.provision "shell", inline: <<-SHELL
+    # add docker repo
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
+
+    # and golang 1.9 support
+    # add-apt-repository ppa:gophers/archive
+    add-apt-repository ppa:longsleep/golang-backports
+
+    # install base requirements
     apt-get update
-    apt-get install -y --no-install-recommends wget curl jq shellcheck bsdmainutils psmisc
+    apt-get install -y --no-install-recommends wget curl jq \
+        make shellcheck bsdmainutils psmisc
+    apt-get install -y docker-ce golang-1.9-go
 
-    wget -qO- https://get.docker.com/ | sh
-    usermod -a -G docker vagrant
-    apt-get autoremove -y
+    # needed for go
+    apt-get install -y git
```

Comment:
this is no longer needed if you're installing from ppa

Reply:
Cool, will update